packages/host/app/lib/file-def-manager.ts

```diff
@@ -315,11 +315,7 @@ export default class FileDefManagerImpl
           throw new Error('File needs a realm server source URL to upload');
         }
 
-        let response = await this.network.authedFetch(file.sourceUrl, {
-          headers: {
-            Accept: 'application/vnd.card+source',
```

Comment:
I anticipate we'll support files other than code source so it makes sense to remove this header IMO